alicloud/resource_alicloud_route_table_attachment.go

```diff
@@ -38,94 +37,84 @@ func resourceAliyunRouteTableAttachmentCreate(d *schema.ResourceData, meta inter
 	client := meta.(*connectivity.AliyunClient)
 	routeTableService := RouteTableService{client}
 
-	args := vpc.CreateAssociateRouteTableRequest()
-	args.RouteTableId = Trim(d.Get("route_table_id").(string))
-	args.VSwitchId = Trim(d.Get("vswitch_id").(string))
+	request := vpc.CreateAssociateRouteTableRequest()
+	request.RouteTableId = Trim(d.Get("route_table_id").(string))
+	request.VSwitchId = Trim(d.Get("vswitch_id").(string))
+	request.ClientToken = buildClientToken(request.GetActionName())
 	if err := resource.Retry(3*time.Minute, func() *resource.RetryError {
-		ar := args
-		_, err := client.WithVpcClient(func(vpcClient *vpc.Client) (interface{}, error) {
+		ar := request
+		raw, err := client.WithVpcClient(func(vpcClient *vpc.Client) (interface{}, error) {
 			return vpcClient.AssociateRouteTable(ar)
 		})
```

Comment:
The line 45 should be `args := *request` and line 47 should be `return vpcClient.AssociateRouteTable(&args)`